cpg_workflows/stages/large_cohort.py

```diff
@@ -1047,6 +1047,24 @@ def queue_jobs(self, cohort: Cohort, inputs: StageInput) -> StageOutput | None:
         joint_included: bool = data_download_config.get('joint_included', False)
 
         jobs = []
+
+        prejob = get_batch().new_job(
+            f'RepartitionFrequenciesTable_{data_type}',
+            (self.get_job_attrs() or {}) | {'tool': HAIL_QUERY},
+        )
+        prejob.image(image_path('cpg_workflows'))
```

Comment:
This should probably be replaced with `config_retrieve(['workflow', 'driver_image'])`

`image_path('cpg_workflows')` is fixed at `cpg_workflows:1.36.25` which might not contain the code you need/want to run